sematic/ee/plugins/external_resource/ray/load_testing/BUILD

```diff
@@ -0,0 +1,30 @@
+load("//bazel:pipeline.bzl", "sematic_pipeline")
```

Comment:
Why not put this in a `test` directory?
`sematic/ee/plugins/external_resource/ray/tests/load_testing`

Reply:
Mainly because I generally see those as being reserved for automated tests, which this is not. But I can move.